plugins/scaffolder-backend-module-azure/src/actions/azure.ts

```diff
@@ -207,13 +207,10 @@ export function createPublishAzureAction(options: {
           : config.getOptionalString('scaffolder.defaultAuthor.email'),
       };
 
-      const auth =
-        ctx.input.token || credentials?.type === 'pat'
-          ? {
-              username: 'notempty',
-              password: ctx.input.token ?? credentials!.token,
-            }
-          : { token: credentials!.token };
+      const auth = {
```

Comment:
Could we add a test for this in `azure.test.ts`, please?

Reply:
Sure. Give me a few hours.

Reply:
Navigate to the folder: `cd plugins/scaffolder-backend-module-azure/src/actions` and then run `yarn test azure.test.ts`

Reply:
@awanlin from what I understand you'd like me to test that the proper format of auth is used when calling `initRepoAndPush`.  From what I can tell of the existing code to support the 2 configs (AppReg and ManagedIdentity) I would need to create multiple suites (describes).  Is that assumption correct and is that cool to have multiple suites in the same test file?

Reply:
Sorry, i figured it out and deleted the comment.  For anyone else reading you can just do yarn test then pick the filter option.

Reply:
Sorry  @awanlin  this may take me a bit longer.  It's been a minute since I've done mocking etc. in jest.  I'll need some time to relearn some things.

Reply:
That's alright, take the time you need. 👍

Reply:
@awanlin i'm running into issues with the no-use-before-define ESLint rule.  My jest .spyOn()s are being hoisted above the import statements (as jest is want to do) but this results in the `no-use-before-define` rule being triggered.  Thoughts?

Reply:
I'm swamped and won't be able to help, testing is a bit of a weak spot for me so I would need to pull this down and play around with it.

Reply:
OK, given @benjdlambert comment, let's leave it 👍